.github/workflows/release-please.yml

```
@@ -0,0 +1,33 @@
+# Release Please automates releases using Conventional Commit messages. After each release it maintains an open PR
+# called `chore(main): release <semver>`, which keeps the CHANGELOG.md up to date along with a changelog.json file.
+# When the release PR is merged, this action will create a Github release. Our docs-site PR workflow will use the
+# contents of the changelog.json file and PR headers to generate release notes for the public docs website.
+# See https://github.com/google-github-actions/release-please-action
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: Release Please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          token: ${{ secrets.RELEASE_PLEASE_TOKEN }}
```

Can we use our github bot token for this?

```suggestion
          token: ${{ secrets.BROWSER_GITHUB_BOT_PAT }}
```